### PR TITLE
apply-dashboards-observability-ns

### PR DESCRIPTION
**Pull request by austincunningham**
## What
https://issues.redhat.com/browse/MGDAPI-1885
The observability operator prometheus is namespace scoped so dashboards have to be created in the same namespace as prometheus. 

## How 
Change the Namespace in the deploy script

## Verification

1. deploy the oo-install branch
2. Set the following environment variables:
   ```bash
   export RHOAM=true
   # This env var is optional. Only set it if you want to view the metrics data using the Grafana dashboard
   export GRAFANA_DASHBOARD=true

   ```
3. Then run this command to deploy the app:
   ```make local/deploy```
4. Confirm that the two workload-web-app dashboards are available in the OO Grafana instance

### CODE REVIEW

**Comment by austincunningham** [comment]
@Boomatang You may be the best person to review this

**Comment by laurafitzgerald** [comment]
The code looks alright to me. I was about to run it when I realised I was uninstalling my cluster. If someone else beats me to it then feel free to take it, otherwise I will verify when I have another installation.

**Comment by Boomatang** [comment]
I can run this. Currently have a cluster running

**Comment by Boomatang** [comment]
I had issue deploying the workload app locally but with @austincunningham help we seen the dashboards in the OO granfana.

/lgtm

**Comment by briangallagher** [comment]
@laurafitzgerald @philbrookes @pmccarthy Can one of ye guys approve this?

**Comment by psturc** [comment]
@Boomatang @laurafitzgerald workload-web-app (master branch) is being used by nightly pipelines. Should we merge this one once all OO-related changes are merged in integreatly-operator repo first?

**Comment by laurafitzgerald** [comment]
@psturc yes good point. Let's hold off so we don't break master. Is it just a managed-api install that's done from master or are there any others? managed-api-multitenant for example?

File: deploy/deploy.sh [hunk 1]
```diff
@@ -136,8 +137,8 @@ oc wait -n $NS --for="condition=Ready" pod -l app=workload-web-app --timeout=120
 if [[ ! -z "${GRAFANA_DASHBOARD}" ]]; then
   echo "Creating Grafana Dashboard for the app"
   if [[ -z "${RHOAM}" ]]; then
-    oc apply -n $NS -f $DIR/dashboard.yaml
+    oc apply -n $OBSERVABILITY_NS -f $DIR/dashboard.yaml
```
Comment (KevFan) [inline deploy/deploy.sh]: Is this change needed for RHMI out of curiousity ? :thinking:

**Review by KevFan — COMMENTED** [review]
Looks good, just have a few questions :thinking: 

Should we include manaully deleting the RHOAM dashboard in the `undeploy` script ?
- https://github.com/integr8ly/workload-web-app/blob/master/deploy/undeploy.sh#L10

I think the dashboard probably wont get removed by the `undeploy` script now that it resides in the observability namespace :thinking:

File: deploy/deploy.sh (hunk 1 above)
Comment (austincunningham) [reply deploy/deploy.sh]: very good question and catch , we don't need this for RHMI

**Comment by KevFan** [comment]
* Verified dashaboard available in OO  grafana using `make local/deploy`
* Verified dashaboard removed  in OO grafana using `make local/undeploy`

/lgtm
